src/views/PulseMessages/index.jsx

```diff
@@ -105,8 +105,7 @@ const getBindingsFromProps = props => {
     whiteSpace: 'nowrap',
   },
   startStopIconSpan: {
-    position: 'fixed',
-    bottom: theme.spacing.double,
+    ...theme.mixins.fab,
```

Comment:
I noticed that 
`    position: 'fixed',
    bottom: theme.spacing.double,`

Is used in some other places in the code. For example in `CreateTaks` view for 'createIconSpanClass'. Do you want me also to fix these?

Reply:
That would be really appreciated :) Feel free to do that in a separate pull-request.